### PR TITLE
Cleaning up README and spec template, add PR template

### PR DESCRIPTION
**Pull request by HowardWolosky**
README updates
* Clarified language of the process
* Enforced a 100-char line wrap

Spec template
* Added Title to template
* Added link to feature pitch
* Added reminder to rename to README.md
* Cleaned up line wrap

PR template
* Ensured there was a link to the feature pitch
* Referenced the workflow process to help prevent unexpected PR's.
* Added reminder to rename to README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,44 +1,66 @@
...
+* **DO NOT** provide feedback on spec documents in the `archive` folder since those features have
+  already been released
+* **DO NOT** submit pull requests that create new specs for ideas that do not have an associated
+  approved feature pitch tracked in [Microsoft/calculator](https://github.com/Microsoft/calculator).
+  To propose a new feature, please follow the Calculator
+  [contribution process](https://github.com/Microsoft/calculator/blob/master/CONTRIBUTING.md)
 
 ### Workflow
- 
-1. Ensure that your pitch has been approved and moved into [pre-production](https://github.com/Microsoft/calculator/projects/1). When features enter this stage, a new working branch in this repo will be created: `feature/<feature>`
```
Comment (grochocki) [inline README.md]: We are using branches for in-progress specs. I would like to make sure this is captured.

File: spec_template.md [hunk 2]
```diff
@@ -1,31 +1,80 @@
...
+Update this to reference the approved feature pitch, as that is where ongoing discussion will occur.
+-->
 
 ### Problem Statement
-<!-- What problem are we trying to solve? Who’s the target audience? Is there a customer need or pain point we need to remedy? Is there a business goal or metric we are trying to improve? Do we have a hypothesis we want to prove or disprove? -->
+<!--
+What problem are we trying to solve? Who’s the target audience? Is there a customer need or pain
+point we need to remedy? Is there a business goal or metric we are trying to improve? Do we have a
+hypothesis we want to prove or disprove?
+-->
```
Comment (grochocki) [inline spec_template.md:16]: We should not manually wrap comments.

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+-->
+
+## YOUR FEATURE TITLE
+<!-- Update with your feature title -->
+
+### Feature Pitch Link
+Microsoft/calculator#<featurePitchIssueNumber>
+<!--
+Change <featurePitchIssueNumber> to be the actual number of the Issue tracking the feature
+pitch in the Calculator repo.
```
Comment (grochocki) [inline .github/pull_request_template.md]: We should not manually wrap comments.

File: spec_template.md [hunk 4]
```diff
@@ -1,31 +1,80 @@
...
 
 ### Evidence or User Insights
-<!-- Why should we do this? Potential sources of data: Feedback Hub, other GitHub issues, other anecdotes from listening to customers in person or online, request from another team, telemetry data, user research, market or competitive research -->
+<!--
+Why should we do this? Potential sources of data: Feedback Hub, other GitHub issues, other anecdotes
+from listening to customers in person or online, request from another team, telemetry data,
+user research, market or competitive research.
```
Comment (grochocki) [inline spec_template.md:22]: Do not manually wrap comments.

**Review by grochocki — CHANGES_REQUESTED** [review]

File: spec_template.md (hunk 4 above)
Comment (HowardWolosky) [reply spec_template.md:22]: On this one I disagree.  Users will be downloading this file and using it as the basis of their spec.  We should prevent the need for horizontal scrolling when they are editing their content.

File: spec_template.md (hunk 2 above)
Comment (HowardWolosky) [reply spec_template.md:16]: On this one I disagree. Users will be downloading this file and using it as the basis of their spec. We should prevent the need for horizontal scrolling when they are editing their content.

File: .github/pull_request_template.md (hunk 3 above)
Comment (HowardWolosky) [reply .github/pull_request_template.md]: Agreed.  You demonstrated in the other PR how manually wrapped comments present a worse experience in the Issue/PR editor window.

**Review by grochocki — DISMISSED** [review]

**Comment by grochocki** [comment]
@HowardWolosky It looks like there are some merge conflicts (my fault for taking too long to accept). Would you mind addressing those so we can merge it in?

**Comment by HowardWolosky** [comment]
> Would you mind addressing those so we can merge it in?

Done.  Re-based on top of master.  I encourage you to re-review the changes as a result.

**Review by grochocki — APPROVED** [review]

**Pull request merged**